java/HealthVault Mobile/HVSample/src/main/java/com/microsoft/hsg/android/hvsample/PersonalImageLoader.java

```diff
@@ -0,0 +1,166 @@
+package com.microsoft.hsg.android.hvsample;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.Callable;
+
+import android.app.Activity;
+import android.app.ActivityManager;
+import android.content.ComponentCallbacks2;
+import android.content.Context;
+import android.content.res.Configuration;
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.support.v4.util.LruCache;
+import android.widget.ImageView;
+import android.widget.Toast;
+
+import com.microsoft.hsg.HVException;
+import com.microsoft.hsg.android.simplexml.HealthVaultApp;
+import com.microsoft.hsg.android.simplexml.client.HealthVaultClient;
+import com.microsoft.hsg.android.simplexml.client.RequestCallback;
+import com.microsoft.hsg.android.simplexml.methods.getthings3.request.ThingRequestGroup2;
+import com.microsoft.hsg.android.simplexml.methods.getthings3.response.ThingResponseGroup2;
+import com.microsoft.hsg.android.simplexml.things.thing.Thing2;
+import com.microsoft.hsg.android.simplexml.things.types.personalimage.PersonalImage;
+import com.microsoft.hsg.android.simplexml.things.types.types.Record;
+
+public class PersonalImageLoader implements ComponentCallbacks2 {
+
+	private ImageLruCache cache;
+	private HealthVaultClient hvClient;
+	private Activity context;
+	
+	private ImageView imageView;
+	
+	public PersonalImageLoader(Activity context,
+			HealthVaultClient client) {
+		ActivityManager am = (ActivityManager) context.getSystemService(Context.ACTIVITY_SERVICE);
```

Comment:
Should use more descriptive variable names, for a sample it will help people follow along better.